spec/system/placements/placements_user_signs_in/non_support_user_signs_in_when_associated_with_a_school_spec.rb

```diff
@@ -11,7 +11,7 @@
   private
 
   def given_a_school_exists
-    @school = create(:placements_school, name: "Hogwarts")
+    @school = create(:placements_school,   name: "Hogwarts")
```

Comment:
is this whitespace change deliberate? (there are a few other similar ones in this PR also)

Reply:
Yeah i must have missed those - should be fixed